core/src/main/scala/cats/syntax/parallel.scala

```diff
@@ -245,6 +245,10 @@ final class ParallelApOps[M[_], A](private val ma: M[A]) extends AnyVal {
 
   def parProduct[B](mb: M[B])(implicit P: Parallel[M]): M[(A, B)] =
     Parallel.parProduct(ma, mb)
+
+  def parReplicateA(n: Int)(implicit P: Parallel[M]): M[List[A]] =
+    if (n <= 0) P.monad.pure(List.empty[A])
+    else Parallel.parSequence(List.fill(n)(ma))
```

Comment:
I guess these two lines can be replaced with just `P.parReplicateA(...)` if you decide to leave `Parallel#parReplicateA` in the `trait Parallel` or with `Parallel.parReplicateA(...)` if you move it into the object. Just to avoid code duplication.